src/ITR-examples/ITR_DV.py

```diff
@@ -1173,10 +1178,10 @@ def recalculate_warehouse_target_year(warehouse_pickle_json, target_year, sector
     if sector:
         # Ensure we have appropriate region for sector...not all benchmarks support all regions for all sectors
         pf_sector_df = df_fundamentals[df_fundamentals.sector.eq(sector)][["sector", "region"]]
-        bm_sector_df = df_ei.drop(columns=df_ei.columns).loc[sector].reset_index("region")
+        # bm_sector_df = df_ei.drop(columns=df_ei.columns).loc[sector].reset_index("region")
```

Comment:
I'm OK leaving these commented out as documentation / starting points for debugging.